glib/src/gstring.rs

```diff
@@ -200,18 +575,36 @@ impl PartialOrd<str> for GString {
 
 impl Eq for GString {}
 
+impl AsRef<GStr> for GString {
+    fn as_ref(&self) -> &GStr {
+        self.as_gstr()
+    }
+}
+
 impl AsRef<str> for GString {
```

Comment:
Maybe `GString` could `Deref` to `&GStr` and `&GStr` to `&str`?

Reply:
Sure. Only caveat, it would break anything that calls `deref` on `GString` manually